solidity/app/contracts/test/TestRouter.sol

```diff
@@ -6,11 +6,7 @@ import "../Router.sol";
 contract TestRouter is Router {
     event InitializeOverload();
 
-    function initialize(address abacusConnectionManager)
-        external
-        override
-        initializer
-    {
+    function initialize(address abacusConnectionManager) external initializer {
```

Comment:
really doesn't matter but I see `abacusConnectionManager` isn't prefixed with an underscore, may be nice just for our style consistency while we're making a related change here